lib/YuiRestClient/App.pm

```diff
@@ -104,4 +106,20 @@ sub tree {
     });
 }
 
+sub richtext {
```

Comment:
perhaps we could add it in alphabetical order as we did in ruby, only for what is contained in YuiRestClient folder, as it will migrated in some point to CPAN.

Reply:
As the methods already shown not in the alphabetical order, adding my methods in the order is not possible right now. The whole class structure should be sorted in the alphabetical way. I'll introduce the change in another PR.